js/index.js

```diff
@@ -68,6 +76,21 @@ SentryCli.prototype.uploadSourceMaps = function(options) {
         '--rewrite'
       ];
 
+      var sourceMapOptions = Object.keys(SOURCEMAPS_OPTIONS).reduce(function(
+        newOptions,
+        sourceMapOption
+      ) {
+        if (options[sourceMapOption]) {
+          newOptions.push(SOURCEMAPS_OPTIONS[sourceMapOption], options[sourceMapOption]);
```

Comment:
This is a bit problematic because of two conditions: 
 - The `--strip-prefix` option allows either a single value or an array. In the latter case, the option name has to be repeated, e.g.: `--strip-prefix foo --strip-prefix bar`. Have a look at `transformIgnore` above.
 - Boolean flags like `--rewrite` or `--validate` do not expect a value, but this code actually invokes `--rewrite true`. This leads to an error, as "true" is interpreted as a path.

Reply:
Agree. This whole function should be extracted into utility function and it should make use of `transformIgnore` (which can be modified to suit our needs).